Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM golang:1.23-alpine3.20 AS builder
+FROM golang:1.23-alpine3.21 AS builder
```

Comment:
if distroless is used as final base container, we can use simple golang:1.23 which is also based on debian

Reply:
Done.